source/change-streams.rst

```diff
@@ -229,6 +229,29 @@ Driver API
     watch(pipeline: Document[], options: Optional<ChangeStreamOptions>): ChangeStream;
   }
 
+  interface Database {
+    /**
+     * Allows a client to observe all changes in a database.
+     * Excludes system collections.
+     * @returns a change stream on all collections in a database
+     * @since 4.0
+     * @see https://docs.mongodb.com/manual/reference/system-collections/
+     */
+    watch(pipeline: Document[], options: Optional<ChangeStreamOptions>): ChangeStream;
+  }
+
+  interface MongoClient {
+    /**
+     * Allows a client to observe all changes in a cluster.
+     * Excludes system collections.
+     * Excludes the "config", "local", and "admin" databases
```

Comment:
Period after "databases", and/or combine this with "Excludes system collections" on the preceding line.

Reply:
done